library/tasks/TagSortBam.wdl

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
When or where do you import this wdl?

Reply:
Hah.. that's embarrassing. Good catch 👍 

I need to import and call this before metrics or the metrics don't work. Will fix.

Reply:
thanks

Reply:
extra `\n`!